cluster/aws/util.sh

```diff
@@ -1007,9 +1007,9 @@ function kube-up {
   # Wait 3 minutes for cluster to come up.  We hit it with a "highstate" after that to
   # make sure that everything is well configured.
   # TODO: Can we poll here?
-  echo "Waiting 3 minutes for cluster to settle"
+  echo "Waiting 1 minute for cluster to settle"
```

Comment:
Yes; the "secret" here is that this really isn't needed at all... most of the time.  It's really in here just so that the sanity check below will pass.  But we've gradually been moving more and more code out of kube-up (in particular CIDR assignment); the only change we make from this point on is to call salt highstate, which should be unnecessary anyway.

But - as you said in another comment - the risk/reward just isn't worth it here.  And it's really nice to be able to do the sanity check.